lib/collections.js

```diff
@@ -26,13 +23,18 @@ var normalizedWords = [];
 
 // The words stored as keys of a dictionary
 var uniqueWordsAsKeys = {};
+for (var i = 0; i < normalizedWords.length; i++) {
+  uniqueWordsAsKeys[normalizedWords[i]] = true;
+}
+//console.log(uniqueWordsAsKeys);
 
 // The keys pulled out into a list
 var uniqueWords = [];
-
-/* your code starts here */
-
-/* your code ends here */
+var i = 0;
+for (var key in uniqueWordsAsKeys) {
+  uniqueWords[i++] = key;
+}
+//console.log(uniqueWords);
```

Comment:
Overall, great job. One comment I would make is that it'd be more idiomatic in this kind of situation to use `push`, since you're continuing to add new items to the end of an array.